.github/workflows/update-oam-defs.yml

```diff
@@ -0,0 +1,94 @@
+          cd ..
+          awk 'FNR==1 && NR>1 { printf("\n%s\n\n","---") } 1' kuma-crds/*.yaml > ~/kuma.yaml
+      - name: Build jsonschema util
+        run: |
+          curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.38.0/install.sh | bash
+          export NVM_DIR="$HOME/.nvm"
+          [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+          nvm install v14.15.3
+          nvm install-latest-npm
+          git clone https://github.com/layer5io/kubeopenapi-jsonschema util
+          cd util
+          npm i; npm i nexe -g
+          make linux
+      - name: Generate Workload definitions
+        run: |
+          mkdir -p ./templates/oam/workloads-$NEW_KUMA_VERSION
+          cd util
+          template='{"apiVersion":"core.oam.dev/v1alpha1","kind":"WorkloadDefinition","metadata":{"@type":"pattern.meshery.io/mesh/workload","meshVersion":"'$NEW_KUMA_VERSION'","meshName":"'$smpMeshName'","k8sAPIVersion":null,"k8sKind":""}}}'
+          crds=$(./kubeopenapi-jsonschema --location ~/kuma.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition")]' -o json --o-filter '$..["spec"]["names"]["kind"]' | jq '.[]')
+          for t in ${crds[@]}; do
+            nameUpper=`echo $t  | tr -d '"'`
+            nameLower=`echo $t  | tr -d '"' | tr '[:upper:]' '[:lower:]'`
+            definitionRef=$(printf %s.kuma.meshery.layer5.io $nameLower)
+            apiVersion=$(./kubeopenapi-jsonschema --location ~/kuma.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..spec.versions[0]' --o-filter "$[].name" -o json | jq '.[]' | tr -d '"')
+            apiGroup=$(./kubeopenapi-jsonschema --location ~/kuma.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..spec' --o-filter "$[].group" -o json | jq '.[]' | tr -d '"')
+            name=`echo $t | tr -d '"' | tr '[:upper:]' '[:lower:]'`
+            definitionRef=$(printf %s.kuma.meshery.layer5.io $name)
+            ./kubeopenapi-jsonschema --location ~/kuma.yaml -t yaml --filter '$[?(@.kind=="CustomResourceDefinition" && @.spec.names.kind=='$t')]..openAPIV3Schema' --o-filter "$[]" -o json | jq '.[] | .version = "'$NEW_KUMA_VERSION'" | .title = "'$(printf %skuma $nameUpper)'"' > ../templates/oam/workloads-$NEW_KUMA_VERSION/$(printf %s.kuma.meshery.layer5io.schema.json $nameLower)
+            echo $template | jq '  ."metadata"."name" = "'$(printf %s.Kuma $nameUpper)'"
+            | ."spec"."metadata"."k8sAPIVersion" = "'$(printf $apiGroup/$apiVersion $apiGroup $apiVersion)'"
+            | ."spec"."metadata"."k8sKind" = "'$nameUpper'"
+            | ."spec"."definitionRef"."name"="'$definitionRef'"' > $dirPath/$nameLower.kuma_definition.json
+          done
+      - name: Cleanup
+        run: |
+          rm -rf util kuma-*
+      - name: Update KUMA version in .env
+        run: |
+          echo "CURRENT_KUMA_VERSION=$NEW_KUMA_VERSION" > .github/.env
```

Comment:
We can delete this as we don't need to take the version from .env that was present in .github.